packages/mattermost-components/rollup.config.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import scss from 'rollup-plugin-scss';
+
+import ts from 'rollup-plugin-ts';
+
+import packagejson from './package.json';
+
+const externals = [
```

Comment:
How do these externals compare to webpack externals? It looks like you don't need to specifically tell it that a dependency is available at `window.Whatever` like you do in webpack.

Of course, I wonder how they'll interact with the web app once we go in the direction of module federation. Hopefully either we can make this work with that or figure out a way to have this build with webpack.